three-card-poker/three-card-poker.js

```diff
@@ -601,10 +589,11 @@ function _hideWagerChips() {
     Object.entries(BETS_AND_WINS).forEach(([key, value]) => {
         delay += 250;
```

Comment:
Even though the delay is a nice effect, I'm finding that the UX is weird when I click "Rebet" too soon after click either "Play" or "Fold" because the games seems to correctly reduce my balance but I interfere with the drawing of the chips on the page, so and they don't reappear unless I add more to that wager diamond.